website/content/v1.1/talos-guides/network/device-selector.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Network Device Selector"
+description: "How to configure network devices by selecting them using hardware information"
+aliases:
+  - ../../guides/device-selector
+---
+
+## Configuring Network Device Using Device Selector
+
+`deviceSelector` is an alternative method of configuring a network device:
+
+```yaml
+machine:
+  ...
+  network:
+    interfaces:
+      - deviceSelector:
+          driver: virtio
+          hardwareAddr: "00:00:*"
+        address: 192.168.88.21
+```
+
+Selector has the following traits:
+
+- qualifiers match a device by reading the hardware information in `/sys/class/net/...`
+- qualifiers are applied using logical `AND`
+- `machine.network.interfaces.deviceConfig` option is mutually exclusive with `machine.network.interfaces.interface`
+- the selector is invalid when it matches multiple devices, the controller will fail and won't create any devices for the malformed selector
+
+The available hardware information used in the selector can be observed in the `LinkStatus` resource (works in maintenance mode):
+
+```yaml
+$ talosctl get links eth0 -o yaml
```

Comment:
super-nit:

```suggestion
# talosctl get links eth0 -o yaml
```

this ways it is rendered as YAML comment, but looks like shell prompt as well :symbols: